packages/plus/src/components/Navigation/components/Item.tsx

```diff
@@ -142,29 +140,29 @@ const StyledContainer = styled(Stack)`
   ${NeutralButtonLink};
   border-radius: ${({ theme }) => theme.radii.default};
 
-  &[data-has-no-expand="false"] {
+  &[data-has-no-expand='false'] {
```

Comment:
Seems like it changed from `"` to `'`, did you install biome correctly on your IDE and removed prettier?

Reply:
I'll check